### PR TITLE
platform list

### PR DESCRIPTION
**Pull request by tpltnt**
Hi,

another attempt at addressing #3506.

Cheers,
tpltnt

### CODE REVIEW

**Comment by kylemcdonald** [comment]
this looks correct, and all the ?s have been replaced. :+1:

File: docs/PLATFORMS.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+# Microsoft Windows #
+
+versions: Win7 - Win10
+
+architectures:
+  * x86
+  * x86_64
+
+notes:
+  * Visual Studio 2012 for x86
```
Comment (arturoc) [inline docs/PLATFORMS.md]: this is not correct it's vs2015 for every platform

File: docs/PLATFORMS.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  * Visual Studio 2015
+  * MSYS2
+  * MinGW / Codeblocks 12.11 for x86
+
```
Comment (arturoc) [inline docs/PLATFORMS.md]: we don't support codeblocks anymore, the msys2 download only has support for x86 by now

File: docs/PLATFORMS.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+openFrameworks currently supports the following platforms:
+
+# Android #
+
+versions: 2.2+
+
+architectures:
+  * armeabi-v7a
+  * x86 (i686)
```
Comment (arturoc) [inline docs/PLATFORMS.md]: i think it's better to specify x86 only, in all the platforms, i686 is correct in the case of android but i386 specified in other places is surely not compatible with the libraries we ship

**Comment by arturoc** [comment]
thanks!

**Pull request merged**